src/data/fin/vec_notation.lean

```diff
@@ -124,6 +124,13 @@ set.ext $ λ y, by simp [fin.exists_fin_succ, eq_comm]
 @[simp] lemma range_empty (u : fin 0 → α) : set.range u = ∅ :=
 set.range_eq_empty _
 
+@[simp] lemma range_singleton (x : α) (u : fin 0 → α) : set.range (matrix.vec_cons x u) = {x} :=
+by rw [range_cons, range_empty, set.union_empty]
+
+@[simp] lemma range_pair (x y : α) (u : fin 0 → α) :
+  set.range (vec_cons x $ vec_cons y u) = {x, y} :=
+by rw [range_cons, range_singleton, set.singleton_union]
```

Comment:
`singleton` and `pair` usually refer to `has_singleton.singleton` and `has_insert.insert`; neither of which appear on the LHS here.

I think maybe `range_fin_two` would be better?